static/swagger-specs/unified-tags.yaml

```diff
@@ -0,0 +1,983 @@
+            property:
+              type: array
+              items:
+                type: string
+                description: "The query that limits the tags retrieved."
+                example: "tagCategoryId=e2b7c656-067b-4413-a366-adde0401df50"
+    createTagBody:
+      type: object
+      properties:
+        name:
+          type: string
+          description: "Required. The name that you want to give the tag."
+          example: "sampleTag"
+        tagCategoryId:
+          type: string
+          description: "Optional. The ID of the tag category that you want the tag to belong to. If not specified, the tag will be created as part of the Uncategorized category."
+          example: "e2b7c656-067b-4413-a366-adde0401df50"
+    updateTagBody:
+      type: object
+      properties:
+        op:
+          type: string
+          description: "The operation for the PATCH call. Only `replace` is supported."
+          example: "replace"
+        path:
+          type: string
+          description: "The path of the field you want to update. Supported values include `/name`, `archived`, and `/tagCategoryId`."
+          example: "name"
+        value:
+          type: string
+          description: "The updated value of the field you want to update."
+          example: "newSampleTag"
+        from:
+          type: string
+          description: "The original value of the field you want to update."
+          example: "sampleTag"
+    updatedTag:
+      type: object
+      allOf: 
+        - $ref: "#/components/schemas/tag"
+        - type: object
+          properties: 
+            name:
+              type: string
+              description: "The updated name of the tag."
+              example: "updatedSampleTag"
+    validateTagsBody:
+      type: object
+      properties:
+        ids:
+          type: array
+          description: "The list of IDs that you're trying to validate."
+          items:
+            type: string
+          example:
+            - "2bd5ddd9-7284-4767-81d9-c75b122f2a6a"
+            - "d113f40c-0097-4626-8d5f-6d5017694453"
+            - "invalid-tag"
+        entity:
+          type: string
+          description: "The entity that is requesting the validation of the tags."
+          example: "{API_KEY}"
+    validateTagsResponse:
+      type: object
+      properties:
+        invalidTags:
+          type: array
+          description: "An array listing the invalid tags."
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+                example: "invalid-tag"
```

Comment:
not a huge issue but the same example schema has both a value in words and a value in the correct ID format. 
I mean you could change this example to be like'sjknd8fd6a8d6a878' 
or alternatively, just a suggestion, you could make it "invalid-tag-ID" for clarity.
![Screen Shot 2024-06-28 at 12 42 14 PM](https://github.com/AdobeDocs/experience-platform-apis/assets/43826746/550d3d58-f8e6-4d6c-a981-5244263791dc)

Reply:
I think having it as `invalid-tag` makes it clear enough that it's a bogus tag, so I don't think that's necessary.